src/components/Navigation/Navigation.css

```diff
@@ -0,0 +1,23 @@
+.nav-container {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+}
+
+@media screen and (max-width: 768px) {
+  .nav-container {
+    padding: 1rem;
+  }
+  .logo {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+  }
+}
+
+.logo > img {
+  max-width: 80%;
+  height: auto;
+  padding: 1rem;
+}
```

Comment:
Is there a way on hover to see what the nav icon will do? When I hover on the logout icon, can it display "logout"? Not anything preventing this from being approved, just an idea for user experience.

Reply:
Hmm, I'm not sure if MUI has built-in methods for this but I'll do some research and add it in - might be able to do it using some kind of tooltip. I tried to use the clearest icons possible, but I agree - I hate when sites throw me a bunch of icons and I have to figure out what they do by clicking them.

Reply:
Agree also. Example: Amazon. Lots of icons. Would not know what it does until you click. I think is good as is. This is our first pass and we're on the final week.

Reply:
MaterialUI made it surprisingly easy to add in, so I went ahead and implemented it. Thank you :)